src/signed-xml.ts

```diff
@@ -323,6 +323,7 @@ export class SignedXml {
          valid(ated). Put simply: if one fails, they are all not trustworthy.
       */
       this.signedReferences = [];
+      // TODO: add this break change here later on for even more security: `this.references = [];`
```

Comment:
I presume "breaking change" is meant here and below.

Reply:
Fixed.